tests/e2e/Steps/actionSteps.js

```diff
@@ -0,0 +1,13 @@
+import {config} from "../wdio.conf";
+export class Steps {
+
+    navigateTo(url) {
+        return browser.url(url);
+    }
+
+    waitForPageisLoaded(url) {
+        browser.waitUntil(() => {
```

Comment:
```
browser.waitUntil(
    () => browser.getUrl() === (config.baseUrl + url),
    config.waitforTimeout
);
```
Note: Arrow function support return statement without word return in case if {} disappeared.